build.cmd

```diff
@@ -26,6 +26,7 @@ set __SkipTestBuild=
 if "%1" == "" goto ArgsDone
 if /i "%1" == "/?" goto Usage
 if /i "%1" == "x64"    (set __BuildArch=x64&&shift&goto Arg_Loop)
+if /i "%1" == "x86"    (set __BuildArch=x86&&shift&goto Arg_Loop)
```

Comment:
Should this also be applied to `build.sh`, like this: [build.sh#L156-L159](https://github.com/dotnet/coreclr/blob/cd84f26ee25e72e29be64addda1ad1c85561af73/build.sh#L156-L159) ?